credit_test1/task1/main.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+#include <string.h>
+
+#define BUFFER_SIZE 257
+#define BASE 3
+
+int main() {
+    char buffer[BUFFER_SIZE];
+    printf("enter the string:\n");
```

Comment:
```suggestion
    printf("enter the binary representation of a number:\n");
```
Недостаточно информативное сообщение
(-0.25)